docs/install-pmm/install-pmm-server/baremetal/docker/env_var.md

```diff
@@ -1,36 +1,164 @@
+| `PMM_INTERFACE_TO_BIND` | Network interface binding |
+
+### Database connections
+
+| Variable | Purpose |
+|----------|----------|
+| `PMM_CLICKHOUSE_*` | ClickHouse connection settings |
+| `PMM_POSTGRES_*` | PostgreSQL connection settings |
+
+### Development & testing
+
+| Variable | Use case |
+|----------|----------|
+| `PMM_TEST_*` | Testing environment settings |
+
+### Supported external variables
+
+- **Grafana**: All `GF_*` variables
+- **VictoriaMetrics**: All `VM_*` variables
+- **Kubernetes**: All `KUBERNETES_*` variables
+- **System**: Standard variables like `HOME`, `PATH`, etc.
+
+## Variables for migrating from PMM v2 to PMM v3
+
+When migrating from PMM v2 to PMM v3, you'll need to update your environment variables to match the new naming convention. This is because PMM v3 introduces several important changes to improve consistency and clarity:
+
+- environment variables now use `PMM_` prefix
+- some boolean flags reversed (e.g., `DISABLE_` → `ENABLE_`)
+- removed deprecated variables
+
+### Examples
+
+```bash
+# PMM v2
+-e DISABLE_UPDATES=true -e DATA_RETENTION=720h
+
+# PMM v3 equivalent
+-e PMM_ENABLE_UPDATES=false -e PMM_DATA_RETENTION=720h
+```
+
+### Migration reference table
+
+??? note "Click to expand migration reference table"
+
+    #### Configuration variables
+    | PMM 2                          | PMM 3                              | Comments                      |
+    |---------------------------------|------------------------------------|------------------------------|
+    | `DATA_RETENTION`                | `PMM_DATA_RETENTION`               |                              |
+    | `DISABLE_ALERTING`              | `PMM_ENABLE_ALERTING`              |                              |
+    | `DISABLE_UPDATES`               | `PMM_ENABLE_UPDATES`               |                              |
+    | `DISABLE_TELEMETRY`             | `PMM_ENABLE_TELEMETRY`             |                              |
+    | `DISABLE_BACKUP_MANAGEMENT`      | `PMM_ENABLE_BACKUP_MANAGEMENT`     | Note the reverted boolean   |
+    | `ENABLE_AZUREDISCOVER`          | `PMM_ENABLE_AZURE_DISCOVER`        |                              |
+    | `ENABLE_RBAC`                   | `PMM_ENABLE_ACCESS_CONTROL`        |                              |
+    | `LESS_LOG_NOISE`                |                                    | Removed in PMM v3            |
+    
+    #### Metrics configuration
+    | PMM 2                          | PMM 3                              | 
+    |---------------------------------|------------------------------------|
+    | `METRICS_RESOLUTION`            | `PMM_METRICS_RESOLUTION`           | 
+    | `METRICS_RESOLUTION_HR`         | `PMM_METRICS_RESOLUTION_HR`        | 
+    | `METRICS_RESOLUTION_LR`         | `PMM_METRICS_RESOLUTION_LR`        | 
+    | `METRICS_RESOLUTION_MR`         | `PMM_METRICS_RESOLUTION_MR`        |    
+    
+    #### Authentication & Percona Platform
```

Comment:
```suggestion
    #### Authentication & Percona Portal
```

Reply:
I remember there was a ticket to rename every mention of "Platform" to "Portal".

Reply:
thank you! I will update this as part of a separate PR to avoid conflicts